shelley/chain-and-ledger/formal-spec/address.tex

```diff
@@ -190,4 +188,13 @@ \section{Addresses}
   \label{fig:defs:addresses}
 \end{figure*}
 
+(NSF: The $pay$ and $stake$ subscripts are not defined or discussed. Are those
+subscripts just for legibility?)
+
+(NSF: $\fun{addr_{rwd}}$ is used throughout, but it seems to be the only such
+``Constructor function made explicit as such. For example, the definitions in
+Figure 8 Helper functions for the Protocol Parameters freely conflate Slot
+values with Epoch values and with KESPeriod values. Is there a reason for this
+special treatment? I feel like I'm maybe missing something.)
```

Comment:
We want to stress that they are different, but contain the same information. What would you suggest?

Reply:
In my opinion, I think it's somewhat tedious but also the most clear to include conversion functions in the declared interface of these abstract types and apply them explicitly throughout. And some natural language explaining their meaning (e.g. what assumptions justifies each conversion).

The other option is to elide the conversion as you have here. But then I wonder if you should also elide it for `$\fun{add_{rwd}}`, or if there's some reason I'm not seeing to treat the two kinds of "conversion" differently.

Reply:
that makes sense to me (explicit conversions), and is in line with what we did for the different between slots and slot durations, so as to not mistake them for each other anywhere. that's not a quick change, so we can make an issue for it.